drs-auth_client/lib/drs/auth_client/client.rb

```diff
@@ -0,0 +1,81 @@
+require "active_support/core_ext/hash/keys"
+require "active_support/core_ext/string/inflections"
+require "faraday"
+
+require "drs/auth_client/models/organisation"
+require "drs/auth_client/models/profile"
+
+module Drs
+  module AuthClient
+    class Client
+      def initialize(auth_token)
+        @auth_token = auth_token
+      end
+
+      def get(path)
+        response = conn.get do |request|
+          request.url path
+          request.headers['Authorization'] = "Bearer #{@auth_token}"
+        end
```

Comment:
Should we check that auth_token is not blank and raise an error if it is blank before attempting to make an API call?